RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc

```diff
@@ -57,10 +62,15 @@ MkFitEventOfHitsProducer::MkFitEventOfHitsProducer(edm::ParameterSet const& iCon
       stripClusterIndexToHitToken_{consumes(iConfig.getParameter<edm::InputTag>("stripHits"))},
       mkFitGeomToken_{esConsumes()},
       putToken_{produces<MkFitEventOfHits>()},
+      usePixelQualityDB_{iConfig.getParameter<bool>("usePixelQualityDB")},
       useStripStripQualityDB_{iConfig.getParameter<bool>("useStripStripQualityDB")} {
```

Comment:
Out of curiosity (and not related specifically to this PR): why "StripStrip" instead of just "Strip" in the name?

Reply:
For good or bad, to be consistent with existing naming (in functionally similar class) in
https://github.com/cms-sw/cmssw/blob/30b5f852ec65ca4f168563873bb210c61e6529f2/RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc#L91-L96